figure_environment.yml

```diff
@@ -0,0 +1,27 @@
+name: phenotype_figures
+channels:
+  - conda-forge
+  - bioconda
+dependencies:
+  - conda-forge::jupyter
```

Comment:
Consider only including the packages that needed to be added with `conda install` (and their specific versions).

Reply:
I'm not sure what you mean? I need all these packages (maybe vscode doesn't?) I use `jupyter lab`

Reply:
I assume you didn't do `conda install` for all of these packages? Ex: doing `conda install scikit-learn` into an empty environment will also install `numpy` (a dependency). So even though all of these packages might be necessary for your code, if someone were to create an environment with this `.yml` file I think not all of these packages need to be listed in the `.yml` file.

Usually, when I create a `.yml` file I try to only include the packages (and their versions) that I specifically installed with `conda install`. I think this makes the environment file more readable and its more clear what you installed to create the environment (with all package versions).

Reply:
ohhh, I gotcha. I never use `conda-install`. I only ever use `conda env create --force --file environment.yml`

Avoiding conda-install prevents me from forgetting that I installed a package and needing to add it to the environment in a separate step.

Reply:
In that case LGTM!